frontend_lib/i18next.scanner/fr/translation.json

```diff
@@ -193,5 +193,12 @@
   "You have reached your storage limit, you cannot add new files": "Vous avez atteint votre limite de stockage, vous ne pouvez pas ajouter de nouveaux fichiers",
   "Attach files": "Joindre des fichiers",
   "Download {{filename}}": "Télécharger {{filename}}",
-  "Translate to {{language}}": "Traduire en {{language}}"
+  "Translate to {{language}}": "Traduire en {{language}}",
+  "Error while adding a tag to the content": "Erreur pendant l'ajout d'un tag au contenu",
+  "Add a tag": "Ajouter un tag",
+  "Create a tag or choose one in the list for this content": "Créez une étiquette ou choisissez-en une dans la liste pour ce contenu",
```

Comment:
Please chose tag or étiquette for the translation

Reply:
fixed